plugins/database/src/general_query.cpp

```diff
@@ -148,7 +149,7 @@ int fkFindName( const char *tableName ) {
             return i;
         }
     }
-    rodsLog( LOG_ERROR, "fkFindName table %s unknown", tableName );
+    log_genquery::error("fkFindName table {} unknown", tableName);
```

Comment:
Need to include the function name at the beginning of the message.
```cpp
log_genquery::error("{}: ...", __func__, ...);
```

Please do the same for the other log messages touched by this PR.